js/bench/decode.ts

```diff
@@ -31,6 +31,12 @@ tiles.forEach(tile => {
   }
 });
 
+let maxTime = 10;
+if (process.env.GITHUB_RUN_ID) {
+  maxTime = 2;
+  console.log(`Running in CI, using smaller maxTime: ${maxTime} seconds`);
```

Comment:
This keeps the script runtime nice and swift, faster than the Java setup required for the encode.jar to be available.